templates/view/viewSelect.html.twig

```diff
@@ -0,0 +1,5 @@
+{{ form_start(form) }}
+<div data-controller="view-selector" data-default-view-update-endpoint="/admin/view/set-default">
```

Comment:
Instead of inserting a route directly in the template (/admin/view/set-default), add a variable to the controller method render array.
Use symfony generateUrl to generate the url from a route name.